core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -664,6 +665,7 @@ Builder populate() {
       defineMethod(SINH, BuiltInMethod.SINH.method, NullPolicy.STRICT);
       defineMethod(TAN, BuiltInMethod.TAN.method, NullPolicy.STRICT);
       defineMethod(TANH, BuiltInMethod.TANH.method, NullPolicy.STRICT);
+      defineMethod(HYPOT, BuiltInMethod.HYPOT.method, NullPolicy.ANY);
       defineMethod(TRUNC, BuiltInMethod.STRUNCATE.method, NullPolicy.STRICT);
```

Comment:
Move HYPOT up so that it is after DEGREES and before IS_INF. The list is approximately ordered, so let's keep it that way.

Reply:
Resolved.